src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp

```diff
@@ -232,6 +239,15 @@ int ExecuteManagedAssembly(
     std::string tpaList;
     AddFilesFromDirectoryToTpaList(clrFilesAbsolutePath, tpaList);
 
+#if defined(__LINUX__)
+    void* coreclrTPLib = dlopen(coreClrTPproviderDllPath.c_str(),RTLD_NOW | RTLD_GLOBAL);
+    if (coreclrTPLib == nullptr)
+    {
+        char *error_msg = dlerror();
+        fprintf(stderr, "Event Tracing is disabled.\n dlopen failed to load tracepoint provider:%s, with error:%s\n",coreClrTpProviderDll,error_msg);
```

Comment:
Yeah, I think we should remove it.  If we're concerned about test coverage - we should have some tests running that make sure that we get events.  That will test the end-to-end pipeline, which would be a great thing, and take care of this check as well.